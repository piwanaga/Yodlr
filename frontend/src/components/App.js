/**App component that renders Routes component */

import React from 'react';
import Routes from './Routes';

const App = () => {
  return (
    <div>
      <Routes />
    </div>
  );
}

export default App;
