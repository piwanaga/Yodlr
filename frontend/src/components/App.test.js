import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import App from './App';

describe('App', () => {
  let Wrapper

  beforeEach(() => {
      Wrapper = ({ children }) => (
          <BrowserRouter>
              {children}
          </BrowserRouter>
      );
  });

  it("passes snapshot test", () => {
    const { asFragment } = render(<App />, {wrapper: Wrapper});
    expect(asFragment()).toMatchSnapshot()
  });
});