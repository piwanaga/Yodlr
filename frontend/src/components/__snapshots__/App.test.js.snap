// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App passes snapshot test 1`] = `
<DocumentFragment>
  <div>
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center"
    >
      <div
        class="MuiGrid-root makeStyles-header-1 MuiGrid-item"
      >
        <h2
          class="MuiTypography-root MuiTypography-h2"
        >
          Yodlr Homepage
        </h2>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-item MuiGrid-justify-xs-center"
      >
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <a
            class="MuiTypography-root MuiTypography-body1"
            href="/register"
          >
            Register
          </a>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item"
        >
          <a
            class="MuiTypography-root MuiTypography-body1"
            href="/admin"
          >
            Admin
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
